src/model.rs

```diff
@@ -109,8 +119,12 @@ pub struct PodJob {
     pub input_stream_path: BTreeMap<String, Input>,
     /// Map output directory to a folder in user data target.
     pub output_stream_path: Blob<FolderOnly>,
-    cpu_limit: f32,
-    memory_limit: u64,
+    /// Maximum allowable cores in fractional cores for the computation.
+    pub cpu_limit: f32,
+    /// Maximum allowable memory in bytes for the computation.
+    pub memory_limit: u64,
+    /// Environment variables to be set in environment.
+    pub env_vars: Option<HashMap<String, String>>,
```

Comment:
Hmm, this is one thing that slip through the cracks. Good addition

Reply:
Create an issue on env_var where it should probably be put in pod and not pod-job, with pod-job ability to override.

Reply:
Add issue to support ARGs also which is a Vec<String> (include as discussion where it should live, either in pod or podjob?)

For env variables same thing